lib/theme_check/language_server/completion_providers/object_completion_provider.rb

```diff
@@ -3,29 +3,32 @@
 module ThemeCheck
   module LanguageServer
     class ObjectCompletionProvider < CompletionProvider
-      def completions(content, cursor)
-        return [] unless (variable_lookup = variable_lookup_at_cursor(content, cursor))
+      def completions(relative_path, line, col)
```

Comment:
(Also not sure in which PR we will tackle the following issue:)

This is working:
![14-07-1fkoq-ggk88](https://user-images.githubusercontent.com/12963280/201620325-99474d25-8a2d-4674-a8a4-5efc9b5f1b5b.jpg)

But completion inside an HTML property does not:
![14-08-8xiju-ssvdh](https://user-images.githubusercontent.com/12963280/201620405-627322f7-0930-4920-a655-774fc34fdf79.jpg)

Reply:
Actually, I believe that the the completion items, in the screenshot above, are coming from the default VS Code language server.

The completion items provided by our language server have different icons for different kinds of completion items:

![image](https://user-images.githubusercontent.com/1079279/203369645-1415233e-b5dd-46ed-b0a1-50ccbd7d7530.png)

Also, I believe it's working inside of HTML attributes. Still, please @Poitrin, feel free to re-open this comment if I'm missing something :)

Reply:
Yeah, I was not able to reproduce the issue I mentioned :)